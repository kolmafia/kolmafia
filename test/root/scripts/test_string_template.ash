string what_do_I_now_know()
{
	return "my ABC";
}

String when = "Now";
string who = 'I';

print(

	`{//Comment
		when

	} {
		who
	} {false?"knew":true?"know":"knee"}\
	  	 

	`

	+

	` { what_do_I_now_know() + "!" }`

);
