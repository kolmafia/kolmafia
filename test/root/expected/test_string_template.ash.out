Now I know my ABC!
